finrl/env/env_stocktrading_cashpenalty.py

```diff
@@ -247,27 +261,36 @@ def step(self, actions):
             self.account_information["reward"].append(reward)
 
             # multiply action values by our scalar multiplier and save
-            actions = actions * self.hmax
+            actions = actions * self.hmax 
+            #buy/sell only if the price is > 0 (no missing data in this particular date 
```

Comment:
is it possible that prices could be lt 0? maybe a dumb question?

Reply:
No it can't be, but in the previous work on incorporating different lengths of data in multi stock trading env, we agreed to do the imputation of missing date with 0. Please see this [notebook ](https://github.com/AI4Finance-LLC/FinRL-Library/blob/master/FinRL_ensemble_multiple_stock_trading_ICAIF_2020.ipynb) to know more

Reply:
ohhhh, I see. 
Ok, well, in that case...
@BruceYanghy @XiaoYangLiu-FinRL we might need to consider making a config variable for missing value (like np.nan or None) so we don't mix up 'no data' for '0 data'.

Reply:
Yes, we previously discussed about adding zeros at the beginning to deal with missing values (not on the market yet).